ceph.spec.in

```diff
@@ -908,6 +908,17 @@ depending on python-ceph should be fixed to depend on python-rados,
 python-rbd, python-rgw or python-cephfs instead.
 %endif
 
+%package grafana-dashboards
+Summary:	The set of grafana dashboards for monitoring purposes
```

Comment:
since 4.6.0 RPM, `BuildArch: noarch` is supported, shall we add it for `ceph-grafana-dashboards` as it architecture independent configurations and presumably we don't need to support ancient rpm based distros .